src/store/view-configs/layers.ts

```diff
@@ -119,18 +118,8 @@ export const useLayerColoringStore = defineStore('layerColoring', () => {
     updateOpacityFunction(viewID, layerID, opFunc);
   };
 
-  const resetToDefault = (
-    viewID: string,
-    dataID: LayerID,
-    image: vtkImageData
-  ) => {
-    const scalars = image.getPointData().getScalars();
-
-    updateColorBy(viewID, dataID, {
-      arrayName: scalars.getName() + dataID,
-      location: 'pointData',
-    });
-    setColorPreset(viewID, dataID, getPreset(dataID));
+  const defaultColorPreset = (viewID: string, layerID: LayerID) => {
```

Comment:
I think this would be better named as `resetToDefault`, or at least have the verb `reset` in the name.

Reply:
Yes, changed to `resetColorPreset`